tests/Badge/Model/UseCase/CreateDownloadsBadgeTest.php

```diff
@@ -58,7 +58,7 @@ public function testShouldCreateDefaultBadgeOnError(): void
             ->will(self::throwException(new \RuntimeException()));
 
         $repository = 'PUGX/badge-poser';
-        $badge = $this->useCase->createDownloadsBadge($repository, 'daily', 'svg');
+        $badge = $this->useCase->createDownloadsBadge($repository, 'daily', 'svg', 'flat');
```

Comment:
You could leave `'svg'` and `'flat'` out since they are determined by the parameter defaults.